src/Features/CSharp/Portable/Debugging/CSharpProximityExpressionsService.cs

```diff
@@ -100,6 +100,8 @@ public CSharpProximityExpressionsService()
             }
         }
 
+        public static IList<string> GetProximityExpressions(SyntaxTree syntaxTree, int position) => Do(syntaxTree, position);
```

Comment:
I could have called the "Do" API but it just didn't feel right. Up for suggestions though